src/Microsoft.ApplicationInsights.AspNetCore/Microsoft.ApplicationInsights.AspNetCore.csproj

```diff
@@ -95,10 +95,11 @@
   </ItemGroup>
 
   <ItemGroup Condition=" '$(TargetFramework)' == 'netstandard2.0' ">
-  <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.0.0" />
-  <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.0.0" />
-  <PackageReference Include="Microsoft.Extensions.DiagnosticAdapter" Version="2.0.0" />
-  <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.0.0" />
+  <PackageReference Include="Microsoft.Extensions.Configuration" Version="2.1.0" />
+  <PackageReference Include="Microsoft.Extensions.Configuration.Json" Version="2.1.0" />
+  <PackageReference Include="Microsoft.Extensions.DiagnosticAdapter" Version="2.1.0" />
+  <PackageReference Include="Microsoft.Extensions.Logging.Abstractions" Version="2.1.0" />
+  <PackageReference Include="Microsoft.Extensions.Logging.ApplicationInsights" Version="2.9.1" />
```

Comment:
This also has Microsoft.ApplicationInsights dependency. So need to see how to avoid conflicts between this and the direct dependency for Microsoft.ApplicationInsights

Reply:
This is not a new issue.Same with taking dependency of Perf/Dep packages. We now need to ensure loggingis also updated with latest base sdk.